src/Api/Rate.php

```diff
@@ -48,7 +55,6 @@ public function getRates() {
       'username' => $this->getUsername(),
       'password' => $this->getPassword(),
       'customerNumber' => $this->getCustomerNumber(),
-      'customerNumber' => $this->con,
     ]);
 
     $canadapost_rates = $request->getRates($originPostalCode, $postalCode, 1);
```

Comment:
The actual request object is provided by the SDK. On the module side:

- We would either call the SDK directly on the shipping method and then this class would not be needed.
- Or, to keep the code more concise and modular, we would provide a service that calls the SDK to fetch the rates.

The latter is better. What we need therefore is to convert this class to a service. No need for properties and a constructor, simply a `getRates` function that accepts the configuration and the shipment as arguments.